assets/js/modules/ads/components/settings/SettingsForm.js

```diff
@@ -20,27 +20,102 @@
+import { useFeature } from '../../../../hooks/useFeature';
+import DisplaySetting from '../../../../components/DisplaySetting';
+const { useSelect } = Data;
 
 export default function SettingsForm() {
+	const iceEnabled = useFeature( 'conversionInfra' );
+	const paxEnabled = useFeature( 'adsPax' );
+
+	const conversionID = useSelect( ( select ) =>
+		select( MODULES_ADS ).getConversionID()
+	);
+
+	const paxConversionID = useSelect( ( select ) =>
+		select( MODULES_ADS ).getPaxConversionID()
+	);
+
+	const extCustomerID = useSelect( ( select ) =>
+		select( MODULES_ADS ).getExtCustomerID()
+	);
+
+	const conversionIDValue =
+		paxEnabled && paxConversionID ? paxConversionID : conversionID;
+
+	const isPaxView = paxEnabled && ( paxConversionID || extCustomerID );
+
 	return (
-		<div className="googlesitekit-ads-settings-fields">
-			<StoreErrorNotices moduleSlug="ads" storeName={ MODULES_ADS } />
-
-			<div className="googlesitekit-setup-module__inputs">
-				<ConversionIDTextField
-					helperText={ __(
-						'The Conversion Tracking ID will help track the performance of ad campaigns for the corresponding account',
-						'google-site-kit'
-					) }
-				/>
+		<Fragment>
+			<div className="googlesitekit-ads-settings-fields">
+				<StoreErrorNotices moduleSlug="ads" storeName={ MODULES_ADS } />
+
+				{ iceEnabled && (
+					<div className="googlesitekit-settings-module__meta-item">
+						<ConversionTrackingToggle />
+					</div>
+				) }
+
+				{ ! isPaxView && (
+					<div className="googlesitekit-setup-module__inputs">
+						<ConversionIDTextField
+							helperText={ __(
+								'The Conversion Tracking ID will help track the performance of ad campaigns for the corresponding account',
+								'google-site-kit'
+							) }
+						/>
+					</div>
+				) }
+
+				{ isPaxView && (
+					<div>
+						<div className="googlesitekit-settings-module__meta-item">
+							<h5 className="googlesitekit-settings-module__meta-item-type">
+								{ __(
+									'Conversion Tracking ID',
+									'google-site-kit'
+								) }
+							</h5>
+							<p className="googlesitekit-settings-module__meta-item-data">
+								{ conversionIDValue === '' &&
+									__( 'None', 'google-site-kit' ) }
+								{ conversionIDValue ||
+									( typeof conversionIDValue ===
+										'undefined' && (
+										<DisplaySetting
+											value={ conversionIDValue }
+										/>
+									) ) }
+							</p>
+						</div>
+						<div className="googlesitekit-settings-module__meta-item">
+							<h5 className="googlesitekit-settings-module__meta-item-type">
+								{ __( 'Customer ID', 'google-site-kit' ) }
+							</h5>
+							<p className="googlesitekit-settings-module__meta-item-data">
+								{ extCustomerID === '' &&
+									__( 'None', 'google-site-kit' ) }
+								{ extCustomerID ||
+									( typeof extCustomerID === 'undefined' && (
+										<DisplaySetting
+											value={ extCustomerID }
+										/>
+									) ) }
```

Comment:
This is kind of a strange bit of code, seems like something we could extract into a helper/component in the future, as it's just a bit unclear as to _why_ we do things this way. Not a blocker, but maybe worth a new issue 🤔